### ISSUE TITLE
Syntax Issue while using `F1-score` as scoring metric

### ISSUE THREAD

**Issue by Selva-sys** [EpistasisLab/tpot#987]
Hi,

I started using tpot package recently and was trying to find the best features that yield good `F1-score`.  But the export file has a syntax issue

## Context of the issue

I ran the `tpotClassifier` and it worked fine. But the exported Pipeline file threw an error when I executed the code in it. It was throwing error as the `steps` attribute was missing from `exported_pipeline` object. This is under the `set_param_recursive` function

## Process to reproduce the issue

1. User creates TPOT instance with `f1` scoring metric 
2. User calls TPOT `fit()` function with training data
3. Checks for score 
4. Export pipeline file generated
5. Run the export pipeline python file
6. Error is thrown at the `set_params_recursive` function as `steps` attribute is not available with `export_pipeline`. Refer screenshot below

## Expected result

Should be able to set the random_state to all steps in pipeline

## Current result

Currently it throws error and I had to comment this line to execute this pipeline. But it worked fine with `accuracy` metric

## Steps attribute missing screenshot

![image](https://user-images.githubusercontent.com/59470011/71721444-0f7fe700-2e60-11ea-94ea-b4a195711688.png)

**Comment by weixuanfu** [comment]
Related to the issue #975. It is fixed in dev branch so far and will be fixed in next version of TPOT. you may install TPOT with patch into an environment via:

```pip install --upgrade --no-deps --force-reinstall git+https://github.com/EpistasisLab/tpot.git@development```

Note:  scikit-learn may need to be updated to 0.22 for using dev branch.

**Comment by weixuanfu** [comment]
The issue was fixed in the latest version of TPOT, thus I closed this issue. Please feel free to reopen this issue if you have any questions.
